docs/sources/style-guide/write-for-developers/index.md

```diff
@@ -177,18 +177,18 @@ ssh-rsa KEY_VALUE USERNAME [_FILENAME_]
 ```
 
 Use descriptive words and phrases when including placeholders, and avoid using X or XXX.
-In Markdown, in front of a placeholder, use an asterisk followed by a backtick. At the end of the placeholder, use a backtick followed by an asterisk. For example, prefer the following raw markdown:
+In Markdown, in front of a placeholder, use an asterisk followed by a backtick and an open bracket. Use a dash instead of an underscore in the placeholder name if necessary. At the end of the placeholder, use an open bracket followed by a backtick and an asterisk. For example, prefer the following raw markdown:
```

Comment:
"prefer"?  Should that be "refer to"?

Reply:
If can see why it might have been written that way originally, but refer to is better. I will fix when your question about bold is resolved.

Reply:
Now says "refer to".